packages/flutter_tools/templates/plugin/pubspec.yaml.tmpl

```diff
@@ -3,6 +3,9 @@ description: {{description}}
 
 flutter:
   plugin:
+    version: 1
```

Comment:
Maybe we should put the version in the root of the pubspec.yaml instead of having it live as a child of plugin. That would require it to be a string, but it would keep versions numbers in sync with what's published in pub.